tests/integration/conftest.py

```diff
@@ -30,6 +31,8 @@
 SATELLITE_PKG_URL = "https://satellite.sat.engineering.redhat.com/pub/katello-ca-consumer-latest.noarch.rpm"
 SATELLITE_PKG_DST = "/usr/share/convert2rhel/subscription-manager/katello-ca-consumer-latest.noarch.rpm"
 
+SYSTEM_RELEASE = os.environ["SYSTEM_RELEASE"]
```

Comment:
Just in case this doesn't work, you can try to us `os.environ.get("SYSTEM_RELEASE")`